client/src/components/TodoCardList/Item/index.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Card, CardBody } from 'mdbreact'
+import { Card, CardBody, Input } from 'mdbreact'
 
-const Item = ({ description }) => (
+const Item = props => (
   <div className="pt-1">
     <Card>
       <CardBody className="text-left">
-        <h5 className="mx-0 my-0">{description}</h5>
+        <Input
+          checked={props.readiness}
```

Comment:
@nikitasakov usually timestamp is used instead of bool in order to have information when a column was set to finished/completed

So, `readiness ` can be changed to `completed_at` with timestamp or date value

And in client code: if completed_at not null then item is completed

Is it clear?

Reply:
@AleksSenkou Ok, but what if we want to mark the card as not completed again? Should we set `null` again?

Reply:
@nikitasakov can we just set value from backend? Will input understand that `null` is false?

Reply:
@AleksSenkou 
<img width="1440" alt="2018-12-11 13 05 38" src="https://user-images.githubusercontent.com/32716983/49793122-7b6b4b00-fd45-11e8-9306-f9ee18a80106.png">
If I write `checked={ null }` it looks like `false` on client.

Reply:
@nikitasakov cool!

And what happens when we use a string with datetime instead?

Reply:
@AleksSenkou As I understand all not empty strings checkbox understand as `true`
<img width="1440" alt="2018-12-11 13 12 34" src="https://user-images.githubusercontent.com/32716983/49793998-9939af80-fd47-11e8-95bd-81795ef0174f.png">

Reply:
@nikitasakov That's exactly what we need. 

For insurance we can add a check:

```js
isCompleted = () =>
  this.state.is_completed != null
```

which will return true if is_completed set to some value